Formula/openssh.rb

```diff
@@ -34,6 +34,9 @@ class Openssh < Formula
   def install
     ENV.append "CPPFLAGS", "-D__APPLE_SANDBOX_NAMED_EXTERNAL__"
 
+    # Ensure sandbox profile prefix is correct.
+    inreplace "sandbox-darwin.c", "@PREFIX@/share/openssh", etc/"ssh"
```

Comment:
It's a patch issue.

We (and MacPorts) actually introduce the problem [here](https://raw.githubusercontent.com/Homebrew/patches/1860b0a74/openssh/patch-sandbox-darwin.c-apple-sandbox-named-external.diff), but whereas [MacPorts replace only the prefix](https://github.com/macports/macports-ports/blob/bff69d3bc0ffe4043eaead2db995275b28adfa0d/net/openssh/Portfile#L63-L67) we install it into `etc/ssh` & so need to swap out a little more.